### ISSUE TITLE
Maybe there is something wrong with the pretrained model file you have offered ,the decompression operation can not be operated,plz check that ,thanks very much '

### ISSUE THREAD

**Issue by westbrookS0** [uzh-rpg/rpg_e2vid#9]

**Comment by supitalp** [comment]
Hello,
I just checked the pretrained model, and it looks fine on my side. Can you try redownloading it?

**Comment by westbrookS0** [comment]
Thanks for your reply.  I downloaded it again, but surely this .tar file can not be unzipped on my side....... i have tried many tools for it... if you don't mind ,could you compress the E2VID_lightweight.pth file to rar form and let me try again...
maybe this is a little impolite,but i really want to use this model ,thank you very much!
(here is my email mf1923068@smail.nju.edu.cn)

**Comment by supitalp** [comment]
The .pth file is not an archive meant to be decompressed by standard file decompressors. Instead, it is a binary PyTorch checkpoint which is meant to be loaded by PyTorch. Please follow the instructions in the README on how to use it.

**Comment by westbrookS0** [comment]
i succesessfully run the awesome model about two days ago, thanks for your help.  respect !
